public/i18n/enumKeyLookups.js

```diff
@@ -0,0 +1,20 @@
+export default {
```

Comment:
What's the difference between the *item* values and *accepting*?

Reply:
Item was an existing string and seemed to be used for filters. When I added missing strings I namespaced with "accepting" because they're from the form field called "accepting". I'll take another pass and make naming more consisten